subprojects/core/src/main/java/org/gradle/deployment/internal/DeploymentRegistry.java

```diff
@@ -51,6 +51,8 @@
 
     /**
      * Behavior when a deployment is out-of-date.
+     * 
+     * @since 4.2
      */
     @Incubating
```

Comment:
should get rid of `@Incubating` since this is an internal type.

Reply:
Done.